metricbeat/_meta/fields.common.yml

```diff
@@ -23,6 +23,11 @@
       description: >
         Event round trip time in microseconds.
 
+    - name: metricset.namespace
```

Comment:
Is this entry meant to be an example? Does it make sense to include it? Seems to me it will cause more confusion.